subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_firmware.c

```diff
@@ -51,6 +51,8 @@ static int image_type = DFU_TARGET_IMAGE_TYPE_ANY;
 static char *fota_path;
 static char *fota_host;
 static int fota_sec_tag;
+static uint8_t percent_downloaded = 0;
+static uint32_t bytes_downloaded = 0;
```

Comment:
No need to zero initialize static variables, _checkpatch.pl_ might even complain about it.

```
If an object that has automatic storage duration is not initialized explicitly, its value is
indeterminate. If an object that has static storage duration is not initialized explicitly,
then:
— if it has pointer type, it is initialized to a null pointer;
— if it has arithmetic type, it is initialized to (positive or unsigned) zero;
— if it is an aggregate, every member is initialized (recursively) according to these rules;
— if it is a union, the first named member is initialized (recursi vely) according to these
rules.
```
[C99 - Chapter 6.7.8](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf)